flytekit/configuration/plugin.py

```diff
@@ -52,6 +53,12 @@ def get_default_image() -> Optional[str]:
     def get_auth_success_html(endpoint: str) -> Optional[str]:
         """Get default success html for auth. Return None to use flytekit's default success html."""
 
+    @staticmethod
+    def get_additional_info_for_execution(
```

Comment:
cc @thomasjpfan maybe we can override this as well?

Reply:
I can not think of something right away.